01-JS-DOM-APIs/Practice1-1/Script/script.js

```diff
@@ -0,0 +1,43 @@
+function load () {
+  console.log('load'); //Only to know the page is loaded
+  let change = document.getElementById("changeclass");
+  change.style.transition = "all 3s ease-in-out";
+  change.style.opacity = 1;
+}
+
+
+function configAjax (methodHttp, url, asyncronic){
+  this.methodHttp = methodHttp;
+  this.url = url;
+  this.asyncronic = asyncronic;
+}
+
+function request(configAjax) {
+  return new Promise(function (resolve, reject) {
+    let req = new XMLHttpRequest();
```

Comment:
try to avoid abbreviations when naming your variables, i know it seems silly but naming a variable *request* instead of *req* will help whoever has to understand your code quickly :)